### PR TITLE
DEV-3897 add subject, test and solution for strange-files

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

File: subjects/strange-files/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+When You create a file with some special character, you will need to `escape` those special characters in order to create that file.
+
+A non-quoted backslash `\` is the escape character. The backslash `\` preceding a character tells the shell to interpret that special character literally.
+
+Example:
+
+If you want to create a file named `foo!\.txt` You have to escape the characters like so:
+
+```console
+User-> code foo\!\\.txt
```
Comment (zamazzal) [inline subjects/strange-files/README.md:30]: `touch` command is better for the creation, in some envs, `vsc` is not installed or `code` alias is not created

**Review by zamazzal — CHANGES_REQUESTED** [review]

File: subjects/strange-files/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/strange-files/README.md]: done

File: subjects/strange-files/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+### Instructions
+
+Create a file `"\?$*'First_file'*$?\"` that will contain `Random text inside!` and **nothing else**.
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/strange-files/README.md]: done

File: subjects/devops/strange-files/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+## Strange Files
+
+We come across files and folders name very regularly. In most of the cases file/folder names are related to the content of the file/folder and starts with numbers or letters. Alpha-Numeric file name are pretty common and very widely used, but this is not the case when we have to deal with file/folder name that has special characters in them.
```
Comment (zamazzal) [inline subjects/devops/strange-files/README.md:3]: this part should be in hints sections

**Review by zamazzal — CHANGES_REQUESTED** [review]

File: subjects/devops/strange-files/README.md (hunk 3 above)
Comment (MSilva95) [reply subjects/devops/strange-files/README.md:23]: done

**Review by zamazzal — APPROVED** [review]

**Pull request merged**